src/unittest/python/serialization/relation_identifier_deserializer_test.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from janusgraph_python.structure.io.graphson.graphson_reader_builder import JanusGraphSONReaderBuilder
+from janusgraph_python.core.datatypes.relation_identifier import RelationIdentifier
+
+
+class TestRelationIdentifierDeserialization(unittest.TestCase):
+
+    def test_relationID_deserialization(self):
+        relation_id = "74q-9n4-b2t-cr4"
+
+        reader = JanusGraphSONReaderBuilder().build()
+
+        rel_id_json = {
```

Comment:
Okay I see your point. So this [seem trivial](https://github.com/JanusGraph/janusgraph-python/pull/39/files/4ab895b8480b0c946d1b5f37a8a1b149b07860e3#r422518898)
But, why would we use `readObject` passing **json string** instead of using `toObject` and passing **dict object** any specific reason?

Reply:
The JSON string is what is sent to JanusGraph Server for evaluation. It's therefore part of the interface used to communicate between the driver and the server. The dictionary representation however is only an internal object of the Python driver that could basically change at any time. Only because the object is correctly created from its dictionary representation doesn't mean that it also works for the JSON string.